src/libs/Navigation/AppNavigator/createCustomStackNavigator/index.js

```diff
@@ -25,6 +29,31 @@ const defaultProps = {
     screenOptions: undefined,
 };
 
+const IFRAME_CONSTANT_KEY = 'iframe-constant-key';
+
+function splitRoutes(routes) {
+    const iframeRoutes = [];
+    const rhpRoutes = [];
+    const otherRoutes = [];
+
+    routes.forEach((route) => {
+        const topMostRouteName = getTopMostCentralPaneRouteName({routes: [route]});
```

Comment:
why don't we just pass `route.state` here?

Reply:
ah I see. if this is the parent route, it gets the top most route of the child route.

otherwise, if this isn't a parent route, we just get undefined.

Reply:
Yes, parts like this are why we are considering doing something with the central pane to make it easier to work with